src/service/programService.js

```diff
@@ -1394,13 +1394,21 @@ async function programCopyCollections(req, response) {
   }
 
   const collections = _.get(data, 'request.collections');
+  // const primaryCollectionId = _.get(data, 'request.collections')[];
+  const collectionIds = [];
+
+  _.forEach(collections, (item) => {
+    collectionIds.push(item.id);
+  });
+
```

Comment:
can we rewrite it to const collectionIds = _.map(collections, 'id') ?

Reply:
Done